docker-compose.yml

```diff
@@ -179,9 +194,12 @@ services:
       <<: *rust-backtrace
     tty: false
     depends_on:
-      - grapl-provision
-      - localstack
-      - redis
+      grapl-provision:
+        condition: service_completed_successfully
+      localstack:
+        condition: service_started
+      redis:
+        condition: service_started
```

Comment:
I was thinking about removing the numerous copies of these we have. I put them in to better match the original, but it seems like noise at this point. Yes, these services require localstack and redis to have started, but:
1. they must have started by the time grapl-provision `service_completed_successfully` in the first place, and
2. `service_started` isn't _really_ the best criteria we want, it's that they're also healthy (though i don't want to add healthchecks for those at the moment).